app/lib/activitypub/activity/create.rb

```diff
@@ -14,7 +14,7 @@ def perform
   private
 
   def create_status
-    return reject_payload! if unsupported_object_type? || non_matching_uri_hosts?(@account.uri, object_uri) || tombstone_exists? || !related_to_local_activity?
+    return reject_payload! if unsupported_object_type? || non_matching_uri_hosts?(@account.uri, object_uri) || tombstone_exists? || !related_to_local_activity? || reject_pattern?
```

Comment:
The activity/create_spec is comically long, and I see this PR adds spec for populating the added setting value and for the validator - might be useful to also add something to create spec showing the reject payload getting done when we trigger via a reject pattern?

Reply:
Added reject pattern `spam` to activity/create_spec.
- No impact on existing tests
- Added test: Triggered by reject pattern, status not created